admin/js/perfecty-push-admin.js

```diff
@@ -35,6 +35,44 @@
 					}
 				}
 			);
+			
+			$( "#form" ).submit(
+				function(e){
+					if ( $( "#perfecty-push-send-notification-schedule-notification" ).is( ":checked" ) ) {
+						// Forse devi fare dei controlli prima
```

Comment:
Could you please use English here? Thank you!

Reply:
Sure, but it was just a note for me I should have removed before committing. It says, more or less: "should I make some checks, before this?". I will give an answer to my question and remove the comment